.github/workflows/ci.yml

```diff
@@ -69,6 +70,7 @@ jobs:
           cd src
           python --version  # just to make sure we're running the right one
           python -m unittest test_typing_extensions.py
+        continue-on-error: ${{ endsWith(matrix.python-version, '-dev') }}
```

Comment:
I think we can remove this now, we're almost in the beta phase and we should aim to support 3.14 from now on.